pkgs/by-name/pa/parinfer-rust-emacs/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  lib,
+  llvmPackages,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "parinfer-rust-emacs";
+  version = "0.4.6";
+
+  src = fetchFromGitHub {
+    owner = "justinbarclay";
+    repo = "parinfer-rust-emacs";
+    rev = "v${version}";
+    hash = "sha256-SNs/75beomxvexfE4+3v/l9Xl5w5SY0EWcORHvRitOw=";
+  };
+
+  cargoHash = "sha256-LmfcY9iR7BGh3dF/raSZTIwburtaQRI3I3XvOZG343M=";
+
+  nativeBuildInputs = [
+    llvmPackages.clang
+    rustPlatform.bindgenHook
+  ];
```

Comment:
Not needed now.

https://github.com/ubolonton/emacs-module-rs/commit/cad972872c61ad2b402a309d2c787b5195e0d503